data/frame_json.go

```diff
@@ -223,7 +583,7 @@ func writeDataFrame(frame *Frame, stream *jsoniter.Stream, includeSchema bool, i
 			return err
```

Comment:
If we go with registered encoder for `data.Frame` then need to set stream.Error here since `jsoniter.ValEncoder.Encode` does not allow to return an error